### PR TITLE
Add naming convention table

### PR DESCRIPTION
**Pull request by isBatak**

### CODE REVIEW

File: React/Naming Cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Naming Cheatsheet
```
Comment (kristian240) [inline React/Naming Cheatsheet.md:1]: We don't need a root heading, handbook will use name of the file as the first heading. We can remove this and change all other levels to `level - 1`. E.g., `### Naming Components -> ## Naming Components`.

**Review by kristian240 — CHANGES_REQUESTED** [review]
You are adding a new chapter, you also need to update the `chapters.yaml` file.

**Comment by fvoska** [comment]
I think we could move this to some framework-agnostic part, this makes sense for Angular as well

**Comment by isBatak** [comment]
> I think we could move this to some framework-agnostic part, this makes sense for Angular as well
@fvoska I was thinking the same thing, but...
What should I do with the **Note** part? 
Do you have something similar to polymorphic components in Angular?

File: Naming Cheat Sheet.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+External link to community maintained cheatsheet:  
+[Naming cheatsheet](https://github.com/kettanaito/naming-cheatsheet/blob/main/README.md)
+
+## Naming Components
+
+### HC/LC/b/p/s Pattern
+There is a useful pattern to follow when naming functions:
+
+```
+high context (HC)? + low context? (LC)? + base + part? + suffix?
```
Comment (isBatak) [inline Naming Cheat Sheet.md:16]:
```suggestion
high context (HC)? + low context (LC)? + base + part? + suffix?
```

**Review by fvoska — APPROVED** [review]

**Comment by fvoska** [comment]
@isBatak this has been opened for a while. I am fine with the proposal, so if there are no additional comments from others, I think we can merge.

**Comment by isBatak** [comment]
@fvoska thx, let me just check it on staging one more time

**Pull request merged**